packages/plugin-core/src/_extension.ts

```diff
@@ -805,12 +821,20 @@ async function showWelcomeOrWhatsNew({
   Logger.info({ ctx, version, previousExtensionVersion });
   switch (extensionInstallStatus) {
     case InstallStatus.INITIAL_INSTALL: {
-      Logger.info({ ctx, msg: "extension, initial install" });
-
+      Logger.info({
+        ctx,
+        msg: `extension, ${
+          isNewInstanceInstall
+            ? "initial install"
+            : "install on new vscode instance"
+        }`,
+      });
       // track how long install process took ^e8itkyfj2rn3
       AnalyticsUtils.track(VSCodeEvents.Install, {
         duration: getDurationMilliseconds(start),
+        isNewInstanceInstall,
```

Comment:
Can we rename this to something like "isSecondaryInstallation" or something like that?  Because even if this is the first ever installation, it's still technically a "new instance install".